### PR TITLE
Update to 0.22

### PR DESCRIPTION
**Pull request by steven2308**

### CODE REVIEW

**Review by ThunderDeliverer — CHANGES_REQUESTED** [review]
Nothing major, just a few minor fixes 😄

File: package.json [hunk 1]
```diff
@@ -19,13 +19,14 @@
   "author": "@rmrk-team",
   "license": "Apache-2.0",
   "dependencies": {
-    "@rmrk-team/evm-contracts": "^0.20.0"
+    "@rmrk-team/evm-contracts": "^0.22.0"
```
[the remark this answers is not in the document]
Comment (steven2308) [reply package.json:22]: It wasn't released at that point but yeah. I think I'll generate another minor soon (after adding emotables) and will update then.

[the remark this answers is not in the document]
Comment (steven2308) [reply package.json:22]: May I merge now? It still looks like changes are requested.

[the remark this answers is not in the document]
Comment (ThunderDeliverer) [reply package.json:22]: Yeah, sorry, it's been a crazy day and I forgot to mark this as approved 🙈

**Pull request merged**